src/Examples.Console/DynatraceExporterExample.cs

```diff
@@ -14,106 +14,101 @@
 // limitations under the License.
 // </copyright>
 
-using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using OpenTelemetry;
 using OpenTelemetry.Metrics;
-using OpenTelemetry.Metrics.Export;
 using OpenTelemetry.Trace;
```

Comment:
Trace :eyes:

Reply:
I think this can be removed, i dont think its used. :eyes:

Reply:
Very strange. In here it doesn't show that's unused, but if I add to the "main" project it does. Funky visual studio. Removed.